frontend/src/utils/api/reviews.js

```diff
@@ -54,7 +54,7 @@ export function getReplies(id) {
 export function postReply(id, form) {
   const headers = new Headers()
   headers.append('Content-Type', 'application/json')
```

Comment:
pretty sure you don't need to do the JSON.stringify I could be wrong, but if you straight up send the body it will work. Given that if you call the parameter body you can just do 

```js
return post(`/review/${id}/replies`, {headers, body })
```

But I'd go further still as headers for all post requests will always be JSON. In which case the API file should be changed to reflect that.

For now just make a note. Need to slightly rework the API.js file.

Reply:
Actually you do need to stringify. Or at least everyone does. I think it is implicitly done anyway, but whatevs.